webapp/publisher/snaps/views.py

```diff
@@ -357,139 +356,73 @@ def redirect_get_register_name():
 @publisher_snaps.route("/register-snap")
 @login_required
 def get_register_name():
-    stores = dashboard.get_stores(flask.session)
-
-    available_stores = logic.filter_available_stores(stores)
-
-    snap_name = flask.request.args.get("snap_name", default="", type=str)
-    store = flask.request.args.get("store", default="", type=str)
-
-    conflict_str = flask.request.args.get(
-        "conflict", default="False", type=str
-    )
-    conflict = conflict_str == "True"
-
-    already_owned_str = flask.request.args.get(
-        "already_owned", default="False", type=str
-    )
-    already_owned = already_owned_str == "True"
-
-    reserved_str = flask.request.args.get(
-        "reserved", default="False", type=str
-    )
-    reserved = reserved_str == "True"
-
-    is_private_str = flask.request.args.get(
-        "is_private", default="False", type=str
-    )
-    is_private = is_private_str == "True"
-
-    context = {
-        "snap_name": snap_name,
-        "is_private": is_private,
-        "conflict": conflict,
-        "already_owned": already_owned,
-        "reserved": reserved,
-        "store": store,
-        "available_stores": available_stores,
-    }
-    return flask.render_template("publisher/register-snap.html", **context)
+    return flask.render_template("store/publisher.html")
 
 
 @publisher_snaps.route("/account/register-snap", methods=["POST"])
 def redirect_post_register_name():
     return flask.redirect(flask.url_for(".post_register_name"), 307)
 
 
-@publisher_snaps.route("/register-snap", methods=["POST"])
+@publisher_snaps.route("/api/register-snap", methods=["POST"])
```

Comment:
Bugs me a little bit that we have the API endpoints mixed up with still remaining Flask views, but I guess it's a byproduct of the migration. Would be good to clean it up at some point.